pkg/controller/job/jobcontroller.go

```diff
@@ -419,7 +424,7 @@ func (jm *JobController) syncJob(key string) error {
 		job.Status.Failed = failed
 
 		if err := jm.updateHandler(&job); err != nil {
-			glog.Errorf("Failed to update job %v, requeuing.  Error: %v", job.Name, err)
+			utilruntime.HandleError(fmt.Errorf("Failed to update job %v, requeuing.  Error: %v", job.Name, err))
```

Comment:
Hmmm... I've noticed we're copying this pattern in most (if not all) our controllers. Seems ok to remove.